### PR TITLE
Added links for youtube, vimeo

### PR DESCRIPTION
**Pull request by vedangj044**
Youtube, Vimeo links in templates/index.html

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -72,7 +72,9 @@ <h2 class="title is-2 is-block-fullhd is-skipped">Links</h2>
             <li>{{ common::link_list_item(label='GitHub Organization', href='https://github.com/drogue-iot') }}</li>
             <li>{{ common::link_list_item(label='Community Chat', href='https://matrix.to/#/#drogue-iot:matrix.org') }}</li>
             <li>{{ common::link_list_item(label='Forum', href='https://discourse.drogue.io') }}</li>
-          </ul>
+	    <li>{{ common::link_list_item(label='Youtube', href='https://www.youtube.com/channel/UC7GZUy2hKidvY6V_3QZfCcA') }}</li>
```
Comment (ctron) [inline templates/index.html:75]:
```suggestion
	    <li>{{ common::link_list_item(label='YouTube', href='https://www.youtube.com/channel/UC7GZUy2hKidvY6V_3QZfCcA') }}</li>
```

**Review by ctron — CHANGES_REQUESTED** [review]
Thanks for thinking about this, and for fixing it!

I found a minor spelling error, added a suggestion.

I also think it would be useful if the Twitter account was added: https://twitter.com/DrogueIoT

**Comment by vedangj044** [comment]
sorry for the inconvenience of re-review.

**Review by ctron — APPROVED** [review]
Awesome. Thanks!

**Pull request merged**